src/pages/AddPersonalBankAccountPage.js

```diff
@@ -130,18 +134,33 @@ class AddPersonalBankAccountPage extends React.Component {
                     shouldShowBackButton
                     onBackButtonPress={Navigation.goBack}
                 />
-                {success ? (
+                {shouldShowSuccess ? (
                     <>
-                        <Text style={[styles.formSuccess, styles.mh5]}>
-                            {success}
-                        </Text>
-                        <View style={[styles.mh5, styles.mb5, styles.flex1, styles.justifyContentEnd]}>
+                        <View style={[styles.pageWrapper, styles.flex1, styles.flexColumn, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                            <Icon
+                                src={Illustrations.TadaBlue}
+                                height={100}
```

Comment:
should we be hardcoding these? Or is there a style that we apply so that it's consistent on different platforms?

Reply:
It seems like we should

Reply:
Do you mean the hardcoded values for the height and width for the icon?

Reply:
Actually I guess we hardcode in a few other places as well. Seems strange, but its probably fine for now

Reply:
Yeah, I was thinking the "correct" height/width for web, might be too big for a smaller form factor. But NAB, looks like it's fine.